projects/ad9656_fmc/zcu102/system_bd.tcl

```diff
@@ -0,0 +1,14 @@
+
+source $ad_hdl_dir/projects/common/zcu102/zcu102_system_bd.tcl
+
+#system ID
+ad_ip_parameter axi_sysid_0 CONFIG.ROM_ADDR_BITS 9
+ad_ip_parameter rom_sys_0 CONFIG.PATH_TO_FILE "[pwd]/mem_init_sys.txt"
+ad_ip_parameter rom_sys_0 CONFIG.ROM_ADDR_BITS 9
+set sys_cstring "sys rom custom string placeholder"
+sysid_gen_sys_init_file $sys_cstring
```

Comment:
please remove line 8 and change line 9 like you see them here:
https://github.com/analogdevicesinc/hdl/blob/c656a2e29bcf3418ec6764fd3d7444d02f405657/projects/ad40xx_fmc/zed/system_bd.tcl#L18-L23

Reply:
solved in v14